docs/standard/frameworks.md

```diff
@@ -98,10 +100,35 @@ Use these guidelines to determine which TFM to use in your app:
 
 #### OS version in TFMs
 
-You can also specify an optional OS version at the end of an OS-specific TFM, for example, `net6.0-ios15.0`. The version indicates which APIs are available to your app or library. It does not control the OS version that your app or library supports at run time. It's used to select the reference assemblies that your project compiles against, and to select assets from NuGet packages. Think of this version as the "platform version" or "OS API version" to disambiguate it from the run-time OS version.
+You can also specify an optional OS version at the end of an OS-specific TFM, for example, `net6.0-ios15.0`. The version indicates which APIs are available to your app or library. It doesn't control the OS version that your app or library supports at run time. It's used to select the reference assemblies that your project compiles against, and to select assets from NuGet packages. Think of this version as the "platform version" or "OS API version" to disambiguate it from the run-time OS version.
 
 When an OS-specific TFM doesn't specify the platform version explicitly, it has an implied value that can be inferred from the base TFM and platform name. For example, the default platform value for iOS in .NET 6 is `15.0`, which means that `net6.0-ios` is shorthand for the canonical `net6.0-ios15.0` TFM. The implied platform version for a newer base TFM may be higher, for example, a future `net8.0-ios` TFM could map to `net8.0-ios16.0`. The shorthand form is intended for use in project files only, and is expanded to the canonical form by the .NET SDK's MSBuild targets before being passed to other tools, such as NuGet.
 
+The following table shows the default target platform values (TPV) for each .NET release.
+
+| .NET version | Platform     | Default TPV |
+|--------------|--------------|-------------|
+| .NET 6       | Android      | 31.0        |
+| .NET 7       | Android      | 33.0        |
+| .NET 8       | Android      | 34.0        |
+| .NET 6       | iOS          | 15.0        |
+| .NET 7       | iOS          | 16.1        |
+| .NET 8       | iOS          | 17.2        |
+| .NET 6       | Mac Catalyst | 15.0        |
+| .NET 7       | Mac Catalyst | 16.1        |
+| .NET 8       | Mac Catalyst | 17.2        |
+| .NET 6       | macOS        | 12.0        |
+| .NET 7       | macOS        | 13.0        |
+| .NET 8       | macOS        | 14.2        |
+| .NET 6       | tvOS         | 15.1        |
+| .NET 7       | tvOS         | 16.1        |
+| .NET 8       | tvOS         | 17.1        |
+| .NET 7       | Tizen        | 7.0         |
+| .NET 8       | Tizen        | 8.0         |
+| .NET 6       | Windows      |             |
```

Comment:
@JonDouglas Any idea what these Windows versions should be?

Reply:
https://github.com/dotnet/sdk/blob/be20b697b83861d0e34ebe0341b65216c1186d0b/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Windows.targets#L32-L34

Should be 7.0 it looks like for all.